tests/sources/test_github_sponsors.py

```diff
@@ -37,3 +37,14 @@ def test_parse_gh_user_gh_sponsors_json(self):
 
         got = self.finder.parse_gh_user_gh_sponsors_json(test_json)
         self.assertEqual(got, 15)
+
+    def test_flux_has_sponsor_this_project(self):
+        self.assertTrue(self.finder.has_sponsor_this_project("FluxML/Flux.jl", 0, []))
```

Comment:
This tests could eventually break if the number of sponsors changes, which seems possible since, IIRC, `Flux.jl` s a pretty popular project. At the least, consider adding a comment that explains this possibility so some future maintainer, even if it is us, is less surprised.

Reply:
Ah good point, will be done in next commit!